components/provisioner/pkg/gqlschema/models_gen.go

```diff
@@ -42,6 +42,7 @@ type AzureProviderConfigInput struct {
 
 type ClusterConfigInput struct {
 	GardenerConfig *GardenerConfigInput `json:"gardenerConfig"`
+	Administrators []*string            `json:"administrators"`
```

Comment:
in KEB userID from ers context is require, I am not sure you should allow for nil fields

Reply:
it's a protection against the situation when during upgrade `Provisioner` run before `KEB`.  In this situation `KEB` doesn't send this param.